spec/features/commands_spec.rb

```diff
@@ -237,6 +237,44 @@
     
   end
 
+  scenario 'david locks an environment and forgets' do
```

Comment:
hey now

Reply:
hah, sorry we use this name a lot in tests.  I think it was the founder originally to be honest.